src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -274,7 +274,7 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
           fieldName !== SendFormFields.FiatAmount
             ? SendFormFields.FiatAmount
             : SendFormFields.CryptoAmount
-        if (inputValue === '') {
+        if (inputValue === '' || Number(inputValue)===0) {
```

Comment:
Not sure about this one, I'm sorry.
- First of all, please check in your console the evaluation of `Number('')`. The purpose of the `Number()` wrapper object (see MDN docs for [Primitive Wrappers](https://developer.mozilla.org/en-US/docs/Glossary/Primitive#primitive_wrapper_objects_in_javascript)) is to cast any value to its number primitive equivalent. Which in the case of an empty string, will evaluate to zero, thus we don't need the left side if going for this solution, as it's effectively duplicated here.
- Zero is a perfectly valid send value, see @willyogo's answer last time I brought this question to the table:

> I think the fact that users must confirm the details of the transaction on their wallet is enough reason for us to not handle this in any special way.
If we get a lot of reports of users accidentally sending 0 value txs, we can reconsider showing a warning, but I'm not aware of these reports and definitely don't think we should block the functionality. Calling a fallback function on a contract is one reason a user might want to send a 0 value tx, but not the only one. Other chains support memos, and it's quite possible a user will want to send a 0 value tx with a memo for example

As a result, `''` (empty field value) and `0` (actually user-input 0) are two very different values, the former which should be handled as empty field with the empty state that goes along with it (as we righly currently do), the latter which should be handled like any other user-input send value.

Merging this would make the send modal handle `0` inputs as empty field values, which is not the expected behavior.